modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractCSharpCodegen.java

```diff
@@ -1644,6 +1666,27 @@ public String toEnumVarName(String name, String datatype) {
         }
     }
 
+    private String adjustNamingStyle(String name)
```

Comment:
minor suggestion: add a docstring explaining what this function does.

Reply:
I have added a doc comment.